docs/how-to/contribute-code.rst

```diff
@@ -354,18 +458,101 @@ After running the test suite, generate a coverage report by running:
 
       (venv) C:\...>tox -e coverage
 
-You can run the test suite and coverage together by including the testing
-environment to run, e.g. ``py,coverage`` or ``py310,coverage``.
+To run the test suite along with this coverage reporting, run:
```

Comment:
This seems a little misleading - `tox p` also runs the test suite and give coverage. The difference is that it *doesn't* do towncrier, docs and pre-commit checks. Feels like there's an "only" or "just" missing here.

Reply:
Oh...this was only to show how you can run tests _and_ get coverage reporting.....as opposed to just running `tox -e coverage` to get the report.

But, tbh, the use-case to run _just_ `tox -e coverage` is a little niche....i mean, it just creates a coverage report from existing coverage files. The most likely use-case for getting coverage is to run the tests over code you _just_ changed and get the coverage report.

So, it might be better to first introduce methods to run tests and get the coverage report....and in note at the end mention that `tox -e coverage` can re-produce a coverage report.